### PR TITLE
Sca teampath integration

### PR DESCRIPTION
**Pull request by SubhadraSahoo**

### CODE REVIEW

File: cxplugin-agent/src/main/java/com/checkmarx/teamcity/agent/CxConfigHelper.java [hunk 1]
```diff
@@ -248,6 +268,11 @@ private static AstScaConfig getScaConfig(Map<String, String> buildParameters, Ma
             scaConfig.setPassword(decrypt(buildParameters.get(SCA_PASSWORD)));
             scaConfig.setUsername(buildParameters.get(SCA_USERNAME));
             scaConfig.setTenant(buildParameters.get(SCA_TENANT));
+            if(buildParameters.get(SCA_TEAMPATH)!= null) {
```
Comment (cx-umesh-waghode) [inline cxplugin-agent/src/main/java/com/checkmarx/teamcity/agent/CxConfigHelper.java]: Use funciton StringUtils.isEmpty

File: cxplugin-agent/src/main/java/com/checkmarx/teamcity/agent/CxConfigHelper.java [hunk 2]
```diff
@@ -65,8 +67,17 @@ public static CxScanConfig resolveConfigurations(Map<String, String> buildParame
...
         ret.setPresetId(convertToIntegerIfNotNull(buildParameters.get(PRESET_ID), PRESET_ID));
         ret.setTeamId(validateNotEmpty(buildParameters.get(TEAM_ID), TEAM_ID));
-
-
+        try {
+        	initializeCommonClient(ret, logger);
+        	commonClient.login();
+			teamPath = commonClient.getTeamNameById(buildParameters.get(TEAM_ID));
+		} catch (Exception e) {
+            logger.error("Failed to get team name by team id: " + e.toString());
```
Comment (cx-umesh-waghode) [inline cxplugin-agent/src/main/java/com/checkmarx/teamcity/agent/CxConfigHelper.java]: Use getMessage and also log 'e'

**Review by cx-umesh-waghode — CHANGES_REQUESTED** [review]

**Review by cx-umesh-waghode — APPROVED** [review]

File: cxplugin-agent/src/main/java/com/checkmarx/teamcity/agent/CxConfigHelper.java (hunk 2 above)
Comment (SubhadraSahoo) [reply cxplugin-agent/src/main/java/com/checkmarx/teamcity/agent/CxConfigHelper.java]: Addressed the code review comment.

File: cxplugin-agent/src/main/java/com/checkmarx/teamcity/agent/CxConfigHelper.java (hunk 1 above)
Comment (SubhadraSahoo) [reply cxplugin-agent/src/main/java/com/checkmarx/teamcity/agent/CxConfigHelper.java]: Addressed the code review comment.

**Pull request merged**